### DISCUSSION TITLE
report_constraint -max_fanout

### DISCUSSION THREAD

**Discussion by vijayank88** [The-OpenROAD-Project/OpenROAD-flow-scripts#1265] [Ideas]
https://ipmart.micro-ip.com/STA/dictionary_538_17/set_max_fanout.html

Can we get similar option in OpenSTA?
```
report_constraint -max_fanout     #command   shows   maximum   fanout   constraint   evaluations
report_port -design_rule  #shows port maximum fanout limits
```
@openroadie @maliberty

**Comment by maliberty** [comment]
Unless these are part of the public SDC standard we can't have identical commands.  We could have similar functionality,
